packages/ui/src/council/components/election/pastElection/PastElectionStats.tsx

```diff
@@ -21,7 +21,7 @@ export const PastElectionStats = ({
   totalVotes,
 }: PastElectionStatsProps) => (
   <Statistics>
-    <StatisticItem title="Ended at">
+    <StatisticItem title="Started at">
```

Comment:
Makes sense! Was my logic error: "Council started at"
```suggestion
    <StatisticItem title="Ended at">
```